.server.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./.babelrc",
+  "presets": [
+    ["@babel/env", {
+      "targets": {
+        "node": "current"
```

Comment:
I didn't want `regenerate-runtime`, so I splitted babelrc. 
But this is required to change `import` to `require`.